mne/viz/evoked.py

```diff
@@ -1778,20 +1778,20 @@ def _set_ylims_plot_compare_evokeds(ax, any_positive, any_negative, ymin, ymax,
 
     fraction = 2 if ax.get_ylim()[0] >= 0 else 3
 
-    if truncate_yaxis is not False:
+    if truncate_yaxis:
         _, ymax_bound = _truncate_yaxis(
             ax, ymin, ymax, orig_ymin, orig_ymax, fraction,
             any_positive, any_negative, truncate_yaxis)
     else:
-        if truncate_yaxis is True and ymin is not None and ymin > 0:
+        if ymin is not None and ymin > 0:
```

Comment:
given above condition, `truncate_yaxis` is guaranteed to be Falsey, so why testing if `is True` here?